x-pack/plugins/lens/public/types.ts

```diff
@@ -103,9 +103,16 @@ export interface TableSuggestion {
  * * `unchanged` means the table is the same in the currently active configuration
  * * `reduced` means the table is a reduced version of the currently active table (some columns dropped, but not all of them)
  * * `extended` means the table is an extended version of the currently active table (added one or multiple additional columns)
+ * * `reorder` means the table columns have changed order, which change the data as well
```

Comment:
out of curiosity, this is not possible yet in current lens?

Reply:
Yeah we were calling these "extended" tables, but that was before wanting to do special handling of these for the XY chart.